autoload/skeleton.vim

```diff
@@ -133,6 +133,9 @@ function! skeleton#DoDefaultReplacements(filename)
   endif
 
   call skeleton#Replace('CURSOR', '')
+  " Substitutions move the cursor's location, so calling this last sets the
+  " cursor's positions to wherever @CURSOR@ is " Equivalent to :cursor()
+  redraw
```

Comment:
I'm not sure this belongs here. There's nothing special about the `CURSOR` substitution except that it's the last substitution performed. How about moving this to the end of `skeleton#Load()` before the `return`?